templates/part.calendarlist.item.php

```diff
@@ -22,56 +22,45 @@
  *
  */
 ?>
-<span class="calendarCheckbox"
+<span class="calendarCheckbox app-navigation-entry-bullet"
 	  ng-click="triggerEnable(item)"
-	  ng-show="item.displayColorIndicator()"
-	  ng-style="{ background : item.calendar.enabled == true ? item.calendar.color : 'transparent' }">
-</span>
-<span class="icon-loading-small pull-left"
-	  ng-show="item.displaySpinner()">
+	  ng-if="item.displayColorIndicator() && !item.calendar.hasWarnings()"
+	  ng-style="{ 'background-color' : item.calendar.enabled == true ? item.calendar.color : 'transparent' }">
 </span>
 <a class="action permanent"
-   ng-class="{'calendar-list-cut-name': item.calendar.isShared() || item.calendar.isPublished() }"
+   ng-class="{
+      'calendar-list-cut-name': item.calendar.isShared() || item.calendar.isPublished(),
+      'icon-error':  item.calendar.hasWarnings()
+   }"
    href="#"
    ng-click="triggerEnable(item)"
-   ng-show="!item.isEditing()"
-   title="{{ item.calendar.displayname }}">
-	<span class="icon icon-error"
-		  ng-if="item.calendar.hasWarnings()"
-		  title="<?php p($l->t('Some events in this calendar are broken. Please check the JS console for more info.')); ?>">
-		&nbsp;&nbsp;&nbsp;&nbsp;
-	</span>
+   title="{{
+      item.calendar.hasWarnings()
+      ? '<?php p($l->t('Some events in this calendar are broken. Please check the JS console for more info.')); ?>'
+      :item.calendar.displayname
+   }}">
```

Comment:
This was never an Error Message or Displayname.

Displayname was always displayed.
Also see @tcitworld's comment about this breaking for french e.g.

Reply:
This issue was not resolved yet and will still cause issues with french locale